src/journey-maps/angular/journey-maps.ts

```diff
@@ -209,6 +210,7 @@ export class SbbJourneyMaps implements OnInit, AfterViewInit, OnDestroy, OnChang
     private _mapLeitPoiService: SbbMapLeitPoiService,
     private _levelSwitchService: SbbLevelSwitcher,
     private _mapLayerFilterService: SbbMapLayerFilter,
+    private _mapOverflowingLabelService: SbbMapOverflowingLabelService,
```

Comment:
This is technically a breaking change. Since I do not expect consumers to actually extend this class, it should be fine.